### ISSUE TITLE
Implicit Parameters are missing from Spec

### ISSUE THREAD

**Issue by natsukagami** [val-lang/specification#20]
From https://github.com/hylo-lang/hylo/pull/1220

**Comment by kyouko-taiga** [comment]
Yes, we have not officially decided what the syntax of implicits should be. I think the discussion will re-open when we get to parts of the standard library that really use this feature. For that we'll have to implement exception and/or unsafe effects first.

**Comment by dabrahams** [comment]
IMO we should not wait to document the syntax we have. Changing syntax in the spec should not be a huge job.
